cdci_data_analysis/analysis/parameters.py

```diff
@@ -391,8 +400,10 @@ def check_value(val, units=None, name=None, par_format=None):
         pass
 
     def reprJSON(self):
-        return dict(name=self.name, units=self.units, value=self.value)
-    
+        reprjson = [dict(name=self.name, units=self.units, value=self.value)]
+        if self.par_format_name is not None:
+            reprjson.append(dict(name=self.par_format_name, units="str", value=self.par_format))
+        return reprjson
```

Comment:
to be clear, json is a string generated from this object. I find sometimes confusion about this, also in the dispatcher, resulting in occasional `try: dat=json.loads(data): except: pass`.